examples/long-running/docker-compose.yml

```
@@ -0,0 +1,24 @@
+version: '3.7'
+
+services:
+
+  script:
+    build:
+      context: .
+    volumes:
+      - ./:/app
+    depends_on:
+      - httpbin
+      - agent
+
+  httpbin:
+    image: kennethreitz/httpbin
+
+  agent:
+    image: datadog/agent:latest
+    volumes:
+    - /var/run/docker.sock:/var/run/docker.sock:ro
+    - /proc/:/host/proc/:ro
+    - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
+    environment:
+    - DD_API_KEY=${DATADOG_API_KEY}
```

The [official agent and docker-compose docs](https://docs.datadoghq.com/integrations/faq/compose-and-the-datadog-agent/) use `DD_API_KEY`; I suggest we do the same.
```suggestion
    - DD_API_KEY=${DD_API_KEY}
```